vizro-core/examples/kpi/app.py

```diff
@@ -0,0 +1,271 @@
+                                color_discrete_sequence=["#1A85FF"],
+                            ),
+                        )
+                    ],
+                ),
+                vm.Container(
+                    title="By Product",
+                    components=[
+                        vm.Graph(
+                            id="bar-product",
+                            figure=bar(
+                                data_frame=df_complaints,
+                                y="Product",
+                                x="Complaint ID",
+                                color_discrete_sequence=["#1A85FF"],
+                            ),
+                        )
+                    ],
+                ),
+                vm.Container(
+                    title="By Channel",
+                    components=[
+                        vm.Graph(
+                            id="bar-channel",
+                            figure=bar(
+                                data_frame=df_complaints,
+                                y="Channel",
+                                x="Complaint ID",
+                                color_discrete_sequence=["#1A85FF"],
+                            ),
+                        )
+                    ],
+                ),
+                vm.Container(
+                    title="By Region",
+                    components=[
+                        vm.Graph(
+                            id="bar-region",
+                            figure=bar(
+                                data_frame=df_complaints,
+                                y="Region",
+                                x="Complaint ID",
+                                color_discrete_sequence=["#1A85FF"],
+                            ),
+                        )
+                    ],
+                ),
+            ],
+        ),
+        vm.Graph(
+            id="line-date",
+            figure=line(
+                data_frame=df_complaints, y="Complaint ID", x="Year-Month Received", color_discrete_sequence=["#1A85FF"]
+            ),
+        ),
+        vm.Graph(
+            figure=pie(
+                data_frame=df_complaints[df_complaints["Company response - Closed"] != "Not closed"],
+                color_discrete_sequence=["#1a85ff", "#7ea1ee", "#adbedc", "#df658c", "#d41159"],
+                custom_order=[
+                    "Closed with explanation",
+                    "Closed without relief",
+                    "Closed with non-monetary relief",
+                    "Closed with relief",
+                    "Closed with monetary relief",
+                ],
+                values="Complaint ID",
+                names="Company response - Closed",
+                title="Closed company responses",
+            )
+        ),
+    ],
+)
+
+
+# TODO: Table-view - Check why date format does not work on `Date Received`
+# TODO: Table-view - Add icons to `Timely` column
+page_table = vm.Page(
```

Comment:
Slightly confusing that this is defined above `page_region` which is not the order in the navigation.